app/controllers/tasks_controller.rb

```diff
@@ -25,6 +31,8 @@ def edit
   # POST /tasks.json
   def create
     @task = Task.new(task_params)
+    @task.user_id = current_user ? current_user.id : 1 #hack to fix test
+    @task.completed = false
```

Comment:
What led you to this approach for not allowing users to complete a task when creating one?